[2024-09-07T00:27:32.208+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-07T00:27:32.227+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_operator_dag_v1.first_task scheduled__2024-09-06T00:00:00+00:00 [queued]>
[2024-09-07T00:27:32.251+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_operator_dag_v1.first_task scheduled__2024-09-06T00:00:00+00:00 [queued]>
[2024-09-07T00:27:32.255+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 3
[2024-09-07T00:27:32.305+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): first_task> on 2024-09-06 00:00:00+00:00
[2024-09-07T00:27:32.345+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=43380) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-07T00:27:32.353+0000] {standard_task_runner.py:72} INFO - Started process 43387 to run task
[2024-09-07T00:27:32.352+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'python_operator_dag_v1', 'first_task', 'scheduled__2024-09-06T00:00:00+00:00', '--job-id', '183', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmp4iza9f1s']
[2024-09-07T00:27:32.374+0000] {standard_task_runner.py:105} INFO - Job 183: Subtask first_task
[2024-09-07T00:27:32.497+0000] {task_command.py:467} INFO - Running <TaskInstance: python_operator_dag_v1.first_task scheduled__2024-09-06T00:00:00+00:00 [running]> on host 626f2e28dec2
[2024-09-07T00:27:32.583+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abhay' AIRFLOW_CTX_DAG_ID='python_operator_dag_v1' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-06T00:00:00+00:00'
[2024-09-07T00:27:32.593+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-07T00:27:32.660+0000] {logging_mixin.py:190} INFO - The sum is 101
[2024-09-07T00:27:32.662+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-07T00:27:32.673+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-07T00:27:32.675+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=python_operator_dag_v1, task_id=first_task, run_id=scheduled__2024-09-06T00:00:00+00:00, execution_date=20240906T000000, start_date=20240907T002732, end_date=20240907T002732
[2024-09-07T00:27:32.752+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-07T00:27:32.826+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-07T00:27:32.833+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
